src/irmin-git/irmin_git.ml

```diff
@@ -170,7 +170,8 @@ struct
         G.Value.blob (G.Value.Blob.of_string str)
     end
 
-    include Content_addressable (GitContents)
+    module Contents = Content_addressable (GitContents)
```

Comment:
The functor's return value can included directly without naming it, as was being done before.